src/js/core.js

```diff
@@ -177,15 +177,10 @@
      */
 
     Core.prototype.editorUpdatePlaceholder = function (el) {
-        var $clone = $(el).clone(),
-            cloneHtml;
+        var contents = $(el).children()
+            .not('.medium-insert-buttons').contents();
 
-        $clone.find('.medium-insert-buttons').remove();
-        cloneHtml = $clone.html()
-            .replace(/^\s+|\s+$/g, '')
-            .replace(/^<p( class="medium-insert-active")?><br><\/p>$/, '');
-
-        if (!(el.querySelector('img, blockquote')) && cloneHtml === '') {
+        if (contents.length === 1 && contents[0].nodeName === 'BR') {
```

Comment:
This is a convention we are using for https://github.com/yabwe/medium-editor
It can avoid browser / (x)html compatibility. Let say your page is xhtml it will return `br` instead of `BR` when you'll call `contents[0].nodeName`, your condition will fail.

See https://developer.mozilla.org/en-US/docs/Web/API/Node/nodeName

```js
var div1 = document.getElementById("d1");
var text_field = document.getElementById("t");

text_field.value = div1.nodeName;
```

> In XHTML (or any other XML format), text_field's value would read "div". However, in HTML, text_field's value would read "DIV", because nodeName and tagName return in upper case on HTML elements in DOMs flagged as HTML documents.